handler.go

```diff
@@ -0,0 +1,34 @@
+package main
+
+import (
+	"fmt"
+	"net/http"
+	"context"
+
+	"github.com/jackc/pgx/v4"
+)
+
+
+
+func Home(w http.ResponseWriter, r *http.Request){
+	fmt.Println("hit home page")
+}
+
+func GetLatestPostings(w http.ResponseWriter, r *http.Request) {
+	ctx := context.Background()
+	conn, _ := pgx.Connect(ctx, "postgresql://admin:quest@localhost:8812/qdb")
```

Comment:
You shouldn't ignore errors from connection objects. You should propagate them up the call stack and handle them.